kotlin-audio/src/main/java/com/doublesymmetry/kotlinaudio/players/BaseAudioPlayer.kt

```diff
@@ -184,9 +184,10 @@ abstract class BaseAudioPlayer internal constructor(private val context: Context
     @CallSuper
     open fun destroy() {
         abandonAudioFocusIfHeld()
+        stop()
         exoPlayer.release()
-        notificationManager.destroy()
         cache?.release()
+        notificationManager.destroy()
     }
```

Comment:
Tried to play with the order of these functions to maybe see if the notification `destroy` method was called too early or too late, but didn't seem to have an effect.